src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/services/UIUsersServiceTest.java

```diff
@@ -67,45 +73,96 @@ void setUp() {
 	}
 
 	@Test
-	void updateUserStatusTest() {
-		ResponseEntity<String> response = service.updateUserStatus(USER2.getId(), false, Locale.ENGLISH);
-		assertEquals(response.getStatusCode(), HttpStatus.OK, "Received an 200 OK response");
+	void createUserTest() throws UIUserFormException {
+		Principal principal = () -> USER1.getFirstName();
+		UserCreateRequest userCreateRequest = new UserCreateRequest(USER2.getUsername(), USER2.getFirstName(),
+				USER2.getLastName(), USER2.getEmail(), USER2.getPhoneNumber(), USER2.getSystemRole().getName(),
+				USER2.getLocale(), false, USER2.getPassword());
+
+		AjaxCreateItemSuccessResponse response = service.createUser(userCreateRequest, principal, Locale.ENGLISH);
+
+		assertEquals(response.getId(), USER2.getId(), "Incorrect user id.");
```

Comment:
this assert has the wrong ordering. It should be `expected`, `actual`, `message` as the params. Please verify the other asserts added to ensure they use this argument ordering.

https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html

Reply:
Please see 9109cf9cc3367a1a7d6a33b075f7ac09fc5e7287.